contracts/UserAuthority.sol

```diff
@@ -90,6 +90,28 @@ contract UserAuthority {
         users[msg.sender].managedUsers.push(userAdr);
     }
 
+    function createAdminAndUser(
```

Comment:
Refactor the previous code, createAdmin, crewate user and create userAndAdmin are using the same code